### ISSUE TITLE
Always getting error from TwitterResult

### ISSUE THREAD

**Issue by devgar** [azyobuzin/tweetust#8]
I'm getting this `invalid response body` error in any request even though the requests are carried out correctly.

For example for this use:

```rust
    let response = TwitterClient::new(auth, handler)
        .statuses().update(status)
        .in_reply_to_status_id(reply_to_status_id)
        .execute();
    match response {
        Ok(res) => println!("Reply id: {}", res.object.id),
        Err(err) => println!("Error on Reply:\n {}", err)
    }
```
I'v got this output:

```
Error on Reply:
 invalid response body: {"created_at":"Wed May 29 16:50:47 +0000 2019","id":1133777718846414848,"id_str":"1133777718846414848","text":"@gardorko \u00bfQuien eres? \u00a7949","truncated":false,"entities":{"hashtags":[],"symbols":[],"user_mentions":[{"screen_name":"gardorko","name":"Red Gar, \ud83c\udf08","id":85646556,"id_str":"85646556","indices":[0,9]}],"urls":[]},"source":"\u003ca href=\"http:\/\/kiwy.band\" rel=\"nofollow\"\u003egarbotko\u003c\/a\u003e","in_reply_to_status_id":1133777695400169474,"in_reply_to_status_id_str":"1133777695400169474","in_reply_to_user_id":85646556,"in_reply_to_user_id_str":"85646556","in_reply_to_screen_name":"gardorko","user":{"id":85646556,"id_str":"85646556","name":"Red Gar, \ud83c\udf08","screen_name":"gardorko","location":"1I\/\u02bbOumuamua","description":"No us\u00e9is nada de lo que yo diga en un juicio.\nPodcaster en @lasagapodcast.","url":null,"entities":{"description":{"urls":[]}},"protected":false,"followers_count":175,"friends_count":586,"listed_count":13,"created_at":"Tue Oct 27 19:51:52 +0000 2009","favourites_count":10961,"utc_offset":null,"time_zone":null,"geo_enabled":true,"verified":false,"statuses_count":5807,"lang":null,"contributors_enabled":false,"is_translator":false,"is_translation_enabled":false,"profile_background_color":"484D48","profile_background_image_url":"http:\/\/abs.twimg.com\/images\/themes\/theme14\/bg.gif","profile_background_image_url_https":"https:\/\/abs.twimg.com\/images\/themes\/theme14\/bg.gif","profile_background_tile":true,"profile_image_url":"http:\/\/pbs.twimg.com\/profile_images\/1012610902766518272\/YaC6Xzco_normal.jpg","profile_image_url_https":"https:\/\/pbs.twimg.com\/profile_images\/1012610902766518272\/YaC6Xzco_normal.jpg","profile_banner_url":"https:\/\/pbs.twimg.com\/profile_banners\/85646556\/1524819697","profile_link_color":"013BA7","profile_sidebar_border_color":"000000","profile_sidebar_fill_color":"EFEFEF","profile_text_color":"333333","profile_use_background_image":true,"has_extended_profile":true,"default_profile":false,"default_profile_image":false,"following":false,"follow_request_sent":false,"notifications":false,"translator_type":"none"},"geo":null,"coordinates":null,"place":null,"contributors":null,"is_quote_status":false,"retweet_count":0,"favorite_count":0,"favorited":false,"retweeted":false,"lang":"es"}
```

**Comment by d6y** [comment]
In case this helps....

I had something [a bit like this](https://github.com/azyobuzin/tweetust/pull/7) a while back. What I did was compare the Rust source models against the JSON and found differences which needed to be fixed up in the source.

For this case, I've had a quick look at the JSON fields in your output and compared it to the models. I noticed two possible differences that might be triggering this:

1. In the JSON the `user` `lang` is `null`. In the _users.rs_ source `lang` is modelled as a `String` rather than an `Option<String>`.

2. In the JSON, the `entities` `user_mentions` contains an `id_str` which isn't represented in `UserMentionEntity` of _entities.rs_.

It _might_ be a case of updating the model to match the JSON.

Hope that's some help.
